examples/aggregate_reporter_param.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
We should not hard-code sensor IDs in the reporter configs.

- we could replace the IDs with the real sensor IDs within the script (maybe using `sed`)
- or we write these JSON configs into the datasources in the first place, from within the script. @Flix6x didn't we talk about that option earlier?

Reply:
Ah okay, thanks

Reply:
@BelhsanHmida maybe add  a `*reporter*.json` to the `.gitignore` if you want to ignore all JSON files dealing with reporters, or alternatively, add `*reporter_param.json` to the `.gitignore` if you still need to keep the `reporter_config.json` files in git. And please hit resolve on this thread once the file is removed from the GH branch.